aws-cli/plan.ps1

```diff
@@ -0,0 +1,32 @@
+$pkg_name="aws-cli"
+$pkg_origin="core"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=('Apache-2.0')
+$pkg_description="The AWS Command Line Interface (CLI) is a unified tool to \
+  manage your AWS services. With just one tool to download and configure, you \
+  can control multiple AWS services from the command line and automate them \
+  through scripts."
+$pkg_upstream_url="https://aws.amazon.com/cli/"
+$pkg_deps=@(
+  "core/python"
+)
+$pkg_bin_dirs=@("Scripts")
+
+function pkg_version {
+  pip search --disable-pip-version-check awscli | %{ if( $_ -match "^awscli \((.+)\)") { $matches[1]; } }
+}
+
+function Invoke-Before {
+  Set-PkgVersion
+}
+
+function Invoke-Prepare {
+  python -m virtualenv "$pkg_prefix"
```

Comment:
I would have thought that this would have required you to `pip install` `virtualenv`. With python3 you should be able to use python's `venv`: `python -m venv $pkg_prefix`

Reply:
I get an error that says the `venv` module does not exist.

Reply:
weird, looking at the packages that is in 3.6 but missing in 3.7. Our build is the same for both so it seems like it is missing from source. investigating...